app/services/description_validator.rb

```diff
@@ -14,13 +14,24 @@ def initialize(csv, bulk_job: false)
   def valid?
     validate_duplicate_headers
     validate_title_headers
+    validate_cell_values
     if @bulk_job
       validate_druid_headers
       validate_druid_rows
     end
     errors.empty?
   end
 
+  def validate_cell_values
+    @csv.each.with_index(2) do |row, i|
+      @headers.excluding('druid').each do |header|
+        location = row['druid'] || "row #{i}"
+        errors << "Value error: #{location} has 0 value in #{header}." if row[header] == '0'
+        errors << "Value error: #{location} has spreadsheet formula error in #{header}." if %w[#NA #REF! #VALUE? #NAME?].include? row[header]
+      end
```

Comment:
The ticket tells us to add the druid to the error message, but single object spreadsheets may not have a druid column, so we will use the row instead in that case.